manifests/package/redhat.pp

```diff
@@ -70,7 +71,7 @@
         }
       }
       'passenger': {
-        if ($facts['os']['name'] in ['RedHat', 'CentOS']) and ($facts['os']['release']['major'] in ['6', '7']) {
+        if ($facts['os']['name'] in ['RedHat', 'CentOS', 'VirtuozzoLinux']) and ($facts['os']['release']['major'] in ['6', '7']) {
```

Comment:
This looks similar to https://github.com/voxpupuli/puppet-nginx/pull/1354/files#diff-2f383a197b1a428561b364755701d60eR146 ; should this line not also have `Oracle` in there?

If that's the case, maybe it'd be an idea to put that snippet into a function, so you only every have to change it in one place.

Reply:
Sadly I don't have an oracle box to test. I don't know which repo is corret. Since it's an RHEL clone `rhel` might actually be correct? IMO we should keep it as is until somebody with an Oracle box tests it / reports back.

Reply:
SGTM!